### ISSUE TITLE
Encoding error on Blame file

### ISSUE THREAD

**Issue by ngreduce** [gitlabhq/gitlabhq#8796]
It should do better than throwing 500.

```
Encoding::CompatibilityError (incompatible character encodings: UTF-8 and ASCII-8BIT):
  app/views/projects/blame/show.html.haml:48:in `block in _app_views_projects_blame_show_html_haml__755271298543089501_12492300'
  app/views/projects/blame/show.html.haml:13:in `_app_views_projects_blame_show_html_haml__755271298543089501_12492300'

Completed 500 Internal Server Error in 6863ms (Views: 88.5ms | ActiveRecord: 34.8ms)
```

**Comment by elboletaire** [comment]
On version 7.7.2 I got that error too but after upgrading to 7.8 seems to be fixed.

What version are you using?

**Comment by ngreduce** [comment]
Latest stable version, when posting issue. Think was 7.7

**Comment by DouweM** [comment]
This has been fixed in 7.8!
